docs/sql-data-sources-json.md

```diff
@@ -155,6 +155,12 @@ Data source options of JSON can be set via:
     <td>Allows leading zeros in numbers (e.g. 00012). If None is set, it uses the default value, <code>false</code>.</td>
     <td>read</td>
   </tr>
+  <tr>
+    <td><code>inferDateType</code></td>
+    <td>None</td>
```

Comment:
Let's change the default value to `false`. This was a mistake from https://github.com/apache/spark/pull/32204, and @itholic is working on it to fix. BTW, seems like CSV option is missing here.

Reply:
done , added for csv also